site-service/src/main/java/uk/ac/ox/ndph/mts/site_service/repository/SiteStore.java

```diff
@@ -0,0 +1,65 @@
+package uk.ac.ox.ndph.mts.site_service.repository;
+
+import org.hl7.fhir.r4.model.Organization;
+import org.hl7.fhir.r4.model.Reference;
+import org.hl7.fhir.r4.model.ResearchStudy;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import uk.ac.ox.ndph.mts.site_service.converter.EntityConverter;
+import uk.ac.ox.ndph.mts.site_service.model.Site;
+
+/**
+ * Implement an EntityStore for Site.
+ */
+@Component
+public class SiteStore implements EntityStore<Site> {
```

Comment:
as said in the EntityStore interface - this class acts as a FHIR service...rename?

Reply:
Taking forward as tech debt.